components/search_engines/brave_prepopulated_engines.h

```diff
@@ -67,11 +66,17 @@ enum BravePrepopulatedEngineID : unsigned int {
   PREPOPULATED_ENGINE_ID_SEARX,            // No longer in defaults (2/2019).
   PREPOPULATED_ENGINE_ID_SEMANTICSCHOLAR,  // No longer in defaults (2/2019).
   PREPOPULATED_ENGINE_ID_STACKOVERFLOW,    // No longer in defaults (2/2019).
-  PREPOPULATED_ENGINE_ID_STARTPAGE_OLD,    // Added to Chromium (2/2025).
-  PREPOPULATED_ENGINE_ID_TWITTER,          // No longer in defaults (2/2019).
-  PREPOPULATED_ENGINE_ID_WIKIPEDIA,        // No longer in defaults (2/2019).
-  PREPOPULATED_ENGINE_ID_WOLFRAMALPHA,     // No longer in defaults (2/2019).
-  PREPOPULATED_ENGINE_ID_YOUTUBE,          // No longer in defaults (2/2019).
+
+  PREPOPULATED_ENGINE_ID_STARTPAGE,  // This ID was used before Chromium added
+                                     // startpage to their prepopulated engines
+                                     // (with id 112). We modify their engine
```

Comment:
nit. upstream uses `113`? https://source.chromium.org/chromium/chromium/src/+/main:third_party/search_engines_data/resources/definitions/prepopulated_engines.json;l=384
```suggestion
                                     // (with id 113). We modify their engine
```

Reply:
Oh wow, they actually changed it from 112 to 113: https://source.chromium.org/chromium/_/chromium/external/search_engines_data/+/f9ec42985da1b80796019c6258317efaa6768cb5. I'll update the comment. Thanks @simonhong!